message_draft_types.go

```diff
@@ -14,6 +14,7 @@ type DraftTemplate struct {
 	BCCList  []*mail.Address
 	Body     string
 	MIMEType rfc822.MIMEType
+	Unread   int
```

Comment:
Prefer `Bool` type here, it's automatically converted from API's 0/1 integer to go's true/false bool

Reply:
Done